ramp-database/rampdb/model/submission.py

```diff
@@ -102,9 +187,16 @@ class Submission(Model):
     # later also ramp_id
     UniqueConstraint(event_team_id, name, name='ts_constraint')
 
-    def __init__(self, name, event_team):
+    # XXX: big change in the database
+    is_sandbox = Column(Boolean, default=False)
+    path_ramp_submissions = Column(String, nullable=False, unique=False)
```

Comment:
It might be more logical to put this new column in the Event table? 
(I don't see a reason to allow it be differ between submissions for one event. I assume the reason to add it to the database in the first place is to allow to have a different directory for submissions *between* events)